stix_shifter_modules/rhacs/configuration/lang_en.json

```diff
@@ -1,24 +1,32 @@
 {
-    "connection": {
-        "host": {
-            "label": "Management IP address or Hostname",
-            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
-        },
-        "port": {
-            "label": "Host Port",
-            "description": "Set the port number that is associated with the Host name or IP"
-        },
-        "help": {
-            "label": "Need additional help?",
-            "description": "More details on the data source setting can be found in the specified link"
-        }
+  "connection": {
+    "host": {
+      "label": "Management IP address or Hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```

Comment:
Please remove "so that IBM Cloud Pak for Security can communicate with it". These should be generic instructions since you would need this regardless of using in CP4S.

Reply:
Updated hostname description as suggested.